testing.md

```diff
@@ -0,0 +1 @@
+This file have to be ignored by .gitignore
```

Comment:
- [ ] As the previous reviewer mentioned, this file should be ignored by git, kindly remove the file from your repo, you can do that by removing the file, committing the changes, then adding the name of the file to .gitignote, and create the file again, this way it will be ignored

Reply:
Hello @Ismailco , I already fixed this issue, and git it's ignoring my file "test.md" now. I only have one more chance to review it, could you tell me if correctly, please?

That was the last point on my observations list.